tests/typings/sync/sync.ts

```diff
@@ -34,7 +34,7 @@ browser.newWindow('https://webdriver.io', {
     windowName: 'some name',
     windowFeatures: 'some features'
 })
-expectType<true | void>(
+expectType<unknown>(
```

Comment:
Why is this `unknown`? Given that the `waitUntil` condition returns `true` I expect this to be `boolean`. What do you think?